apps/relayer/src/transaction/transaction.service.ts

```diff
@@ -63,10 +75,10 @@ export class TransactionService implements OnModuleInit, OnModuleDestroy {
         const txHash = await result.getHash()
         this.watchTransaction(txHash)
         return Ok(txHash)
-      } catch(e) {
+      } catch (e) {
         if (tries === 3) {
           this.logger.error({
-            message: "Failed to send transaction",
```

Comment:
Let's log this as an event as well SendTransactionFailure with `destination`  as a pram.